babel.config.js

```diff
@@ -12,7 +12,9 @@ module.exports = api => {
 
     ignore,
 
-    presets: ['@babel/preset-react', '@babel/preset-env'],
+    presets: ['@babel/preset-react', !esm && '@babel/preset-env'].filter(
```

Comment:
If @babel/preset-env, I'm concerned that we cut it out when doing esm builds.  The problem is the same with cutting out the object rest spread plugin: we make it so consumers must transpile with babel to get the code browser ready.  We instead want it all browser-ready, minus the esm format, and webpack will take care of removing the module format and making it browser ready.

I think this will mean finding out what plugins @babel/preset-env is pulling in for our current env (probably driven by our browserlist?) and then adding those babel plugins specifically here.  

What do you think, Edward?

Reply:
Fair I'll add a conditional on this 
```
{
  "targets": {
    "esmodules": true
  }
}
```